operators/src/source/gdal_source.rs

```diff
@@ -75,131 +78,156 @@ pub trait GdalDatasetInformation {
 
 #[derive(Debug, Serialize, Deserialize, Clone)]
 pub struct JsonDatasetInformationProvider {
-    pub time: TimeIntervalProvider,
-    pub tile: TileGridProvider,
+    pub dataset_information: JsonDatasetInformation,
+    pub raster_data_root: &'static str,
```

Comment:
Unfortunately, I doubt that it will be static, even if it comes from our config.

Reply:
changed it to PathBuf similar to ExecutionContext